commit 207bc1198e3a23ca12af51638781bfd0b50fae6f

--- origsrc/gnome-contacts-3.14.2/src/contacts-types.vala	2014-05-29 16:18:35.000000000 -0500
+++ src/gnome-contacts-3.14.2/src/contacts-types.vala	2015-11-16 02:27:54.859597900 -0600
@@ -47,7 +47,7 @@ public class Contacts.TypeSet : Object
   // Map from display name to TreeIter for all custom types
   private HashTable<string, TreeIter?> custom_hash;
 
-  public ListStore store;
+  public Gtk.ListStore store;
   private TreeIter other_iter;
   private TreeIter custom_iter;
 
@@ -56,7 +56,7 @@ public class Contacts.TypeSet : Object
     vcard_lookup_hash = new HashTable<unowned string, GLib.List<InitData*> > (str_hash, str_equal);
     custom_hash = new HashTable<unowned string, TreeIter? > (str_hash, str_equal);
 
-    store = new ListStore (2,
+    store = new Gtk.ListStore (2,
 			   // Display name or null for separator
 			   typeof(string?),
 			   // Data for standard types, null for custom
